api/report-templates/nameRequest.html

```diff
@@ -19,8 +19,7 @@
                   </div>
                   <div class="report-type-desc">
                       <span class="support-contact-details">
-                         <span class="contact-number">Canada & U.S. Toll Free: 1-877-370-1033</span>
-                        <a href="www.bcregistry.ca/namerequest" target="_blank">www.bcregistry.ca/namerequest</a>
+                           <a href="www.bcregistry.ca/namerequest" target="_blank">www.bcregistry.ca/namerequest</a>
```

Comment:
Can this be a variable? We will replace this url soon.

Reply:
I added NAME_REQUEST_URL to namex vault in 1password, and replaced mentions of www.bcregistry.ca/namerequest with env variable.

Reply:
We already had the URL in web-url vault.

![image](https://github.com/bcgov/namex/assets/44277752/88cf9e26-cfc7-4a51-a5c7-50b001718402)

Reply:
ah.. was searching for the old url. i updated vault.json to pull the variable